airflow/lineage/__init__.py

```diff
@@ -104,7 +104,7 @@ def prepare_lineage(func: T) -> T:
     * "auto" -> picks up any outlets from direct upstream tasks that have outlets defined, as such that
       if A -> B -> C and B does not have outlets but A does, these are provided as inlets.
     * "list of task_ids" -> picks up outlets from the upstream task_ids
-    * "list of datasets" -> manually defined list of data
+    * "list of assets" -> manually defined list of asset
```

Comment:
I believe this refers to [OpenLineage datasets](https://openlineage.io/docs/spec/object-model/), not Airflow’s, so we should keep it as-is.

Reply:
(This is one of the many reasons why we want to rename, fwiw.)

Reply:
ah, got it. I misunderstood this one. 🤦‍♂️ Let me update it.